moveit_ros/planning/planning_request_adapters_plugin_description.xml

```diff
@@ -30,4 +30,9 @@
     </description>
   </class>
 
+  <class name="default_planner_request_adapters/AddIterativeSplineParameterization" type="default_planner_request_adapters::AddIterativeSplineParameterization" base_class_type="planning_request_adapter::PlanningRequestAdapter">
```

Comment:
I'm in favor or reverting #382 (this [PR](https://github.com/ros-planning/moveit/pull/443)) so you shouldn't need to worry about colliding with it. Still, I like your more detailed naming of using "iterative"